src/components/Instrument.tsx

```diff
@@ -150,15 +165,26 @@ export const Instrument: React.FC<Element> = ({ uuid, name, element, x, y, width
                 </div>
                 <div className="px-4 py-1.5 flex gap-3 items-center">
                     <h4 className="font-medium">{name}</h4>
-                    <button className="ml-auto" onClick={refresh}>
-                        <HiRefresh className="cursor-pointer active:text-silver-500" size={22} />
-                    </button>
-                    <ToggleInput onClick={handleWatch} />
+                    <ToggleInput tooltip="Toggle Interaction Mode" onClick={() => setInteractable(!interactable)} />
+
+                    <Tippy content="Refresh Instrument" appendTo={document.body}>
+                        <button className="ml-auto" onClick={refresh}>
+                            <HiRefresh className="cursor-pointer active:text-silver-500" size={22} />
+                        </button>
+                    </Tippy>
+
+                    <ToggleInput tooltip="Toggle Auto-Refresh" onClick={handleWatch} />
                 </div>
             </div>
             <div className="absolute w-full h-full box-content border-2 border-silver-700 bg-black">
                 <InstrumentFrame ref={iframeRef} name={name} width={width} height={height} />
             </div>
+            <div
+                className={classNames(
+                    'absolute w-full h-full box-content border-2 border-silver-700 bg-transparent',
+                    { 'pointer-events-none': interactable },
+                )}
+            />
```

Comment:
Wait I just noticed this, what's this for? Can't we can just toggle `pointer-events-none` directly on the container (lines 145 - 156)?

Reply:
Nope, then you won't be able to click any buttons on the instrument. Putting it at line 179 works

Reply:
I could've sworn I had that working before. I'll take a closer look when I get home.